api/src/main/resources/workbench-api.yaml

```diff
@@ -8091,6 +8068,30 @@ definitions:
         description: item count
         type: integer
         format: int64
+  CriteriaSearchRequest:
```

Comment:
CriteriaSurveyRequest does not capture the surveyName or limit parameters. I am guessing these not used in general?